docs/querying/basics.md

```diff
@@ -218,30 +251,31 @@ For example, the following expression returns the value of
 Note that the `@` modifier always needs to follow the selector
 immediately, i.e. the following would be correct:
 
-    sum(http_requests_total{method="GET"} @ 1609746000) // GOOD.
+    sum( http_requests_total{method="GET"} @ 1609746000 ) // GOOD.
 
 While the following would be *incorrect*:
 
-    sum(http_requests_total{method="GET"}) @ 1609746000 // INVALID.
+    sum( http_requests_total{method="GET"} ) @ 1609746000 // INVALID.
 
 The same works for range vectors. This returns the 5-minute rate that
 `http_requests_total` had at `2021-01-04T07:40:00+00:00`:
 
     rate(http_requests_total[5m] @ 1609746000)
 
-`@` modifier supports all representation of float literals described
-above within the limits of `int64`. It can also be used along
-with `offset` modifier where the offset is applied relative to the `@`
-modifier time irrespective of which modifier is written first.
-These 2 queries will produce the same result.
+`@` modifier supports all representation of numeric literals described
+above within the limits of `int64`. This can be used with the `offset` modifier, 
+where the offset is applied relative to the `@` modifier time.  The results are the
+same irrespective of which modifier is written first.
+
+These two queries will produce the same result.
 
     # offset after @
     http_requests_total @ 1609746000 offset 5m
     # offset before @
     http_requests_total offset 5m @ 1609746000
 
 This modifier is disabled by default since it breaks the invariant that PromQL
-does not look ahead of the evaluation time for samples. It can be enabled by setting
+does not look ahead of the evaluation time for samples. This can be enabled by setting
```

Comment:
Why repeating this?

Reply:
Thought:

Replacing the word "This" in place of "It" continues the use of the established subject.

Reply:
Yes, you are right, "it" is better as it is a continuation of the previous subject.